setup.cfg

```diff
@@ -1,4 +1,62 @@
 [metadata]
 name = detection_rules
+description = Detection Rules is the home for rules used by Elastic Security. This repository is used for the development, maintenance, testing, validation, and release of rules for Elastic Security’s Detection Engine.
+long_description = file: README.md
+version = 8.5.0
```

Comment:
is this just going to match~ with etc/packages.yml:name?

Reply:
yup. pulls from that file now.